### PR TITLE
Refactor Show callout sample to use composable geoview

### PR DESCRIPTION
**Pull request by TADraeseke**
## Description

Refactor "Show callout" sample to use composable geoview

## What To Review

All changes in the PR

## How to Test

Run the sample on the sample viewer or the repo.

### CODE REVIEW

**Review by shubham7109 — APPROVED** [review]
Looks good to approve 👍 . The only change request I would have is to add the GeoView Compose note on the Additional Information section in the readme, and update the relevant metadata/tags.

File: show-callout/src/main/java/com/esri/arcgismaps/sample/showcallout/screens/MainScreen.kt [hunk 1]
```diff
@@ -43,14 +49,37 @@ fun MainScreen(sampleName: String, application: Application) {
...
+                                shapes = CalloutDefaults.shapes(
+                                    calloutContentPadding = PaddingValues(4.dp)
+                                ),
+                                colorScheme = CalloutDefaults.colors(
+                                    backgroundColor = MaterialTheme.colorScheme.background,
+                                    borderColor = MaterialTheme.colorScheme.outline
+                                )
+                            ) {
+                                // Callout content:
+                                Column {
```
Comment (hud10837) [inline show-callout/src/main/java/com/esri/arcgismaps/sample/showcallout/screens/MainScreen.kt:74]: What purpose does this column serve?

File: show-callout/README.md
Comment (hud10837) [inline show-callout/README.md:1]: I think the `How it works` section should be updated here

**Review by hud10837 — CHANGES_REQUESTED** [review]
A couple thoughts here

File: show-callout/README.md [hunk 2]
```diff
@@ -16,9 +16,7 @@ Tap anywhere on the map. A callout showing the WGS84 coordinates for the tapped
 
 1. When the user taps, get the tapped location(map point) from the `SingleTapConfirmedEvent`.
 2. Project the point's geometry to WGS84 using `GeometryEngine.projectOrNull(mapPoint, SpatialReference.wgs84())`.
-3. Create a new Android TextView object and set its text to the coordinate string from the point.
```
Comment (hud10837) [inline show-callout/README.md:19]: I think you also need a step for adding the callout to the MapView content, passing in the location and callout content

**Review by hud10837 — CHANGES_REQUESTED** [review]

File: show-callout/README.md (hunk 2 above)
Comment (TADraeseke) [reply show-callout/README.md:19]: I've reworded and broken into more detailed steps. Thanks!

**Review by hud10837 — APPROVED** [review]
Thanks for making those changes!

**Pull request merged**